kolibri/plugins/learn/assets/src/views/content-points/index.vue

```diff
@@ -88,6 +88,7 @@
     height: 0
     left: 10%
     top: 50%
+    z-index: 1000
```

Comment:
Stick with the z-index values spec'd in [Material Design](https://material.io/guidelines/material-design/elevation-shadows.html) In this case it would be `24` for a dialog.